api/src/main/java/javax/enterprise/event/TransactionPhase.java

```diff
@@ -45,13 +48,23 @@
      * <p>
      * Identifies a before completion observer method, called during the before completion phase of the transaction.
      * </p>
+     * <p>
+     * It will also get invoked if there is no transaction in progress, or the transaction is in progress,
```

Comment:
I'd say it's OK as long as we talk about methods (which we do - it's OM here).
But we might also use something like "get notified" if that sounds better?